### ISSUE TITLE
setBorderColor, setBorderStyle does not work.

### ISSUE THREAD

**Issue by Twood1130** [dbzhang800/QtXlsxWriter#170]
I am attempting to set the border style, and color of a cell to that of another cell, and nothing appears to happen.

**Comment by Twood1130** [comment]
auto xrayFormat = xray->cellAt(j,column)->format();

//reverse the border style of the cell
pinside->cellAt(j,i)->format().setLeftBorderStyle(xrayFormat.rightBorderStyle());
pinside->cellAt(j,i)->format().setRightBorderStyle(xrayFormat.leftBorderStyle());

**Comment by Twood1130** [comment]
Everything compiles fine, but those lines do not change anything in the document. Everything else I am doing in the code executes fine.

**Comment by VSRonin** [comment]
This library uses the get/edit/set paradigm:
```
auto editFormat = pinside->cellAt(j,i)->format();
editFormat.setLeftBorderStyle(xrayFormat.rightBorderStyle());
editFormat.setRightBorderStyle(xrayFormat.leftBorderStyle());
pinside->cellAt(j,i)->setFormat(editFormat);
```

**Comment by Twood1130** [comment]
setFormat is not a member of Cell.

**Comment by VSRonin** [comment]
Yep, my fault, didn't check the docs. You have to use `pinside->write` passing the format as parameter
